spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -124,6 +130,7 @@ def _assigns
             end
           end
           include local
```

Comment:
Quite surprised, but defining those methods direcly in the class breaks the spec.

Reply:
I'm not sure what this refers to

Reply:
I've tried to:
```ruby
Class.new do
  def received
    @received ||= []
  end
  # ...
```
but this doesn't work.

Reply:
Doesn't work as in?

Reply:
No I meant how doesn't it work, how does:
```
Class.new do
  def received
    @received ||= []
  end
end
```
Not have the same effect as above? What is the error it causes.

Reply:
It's this:
```
  4) RSpec::Rails::ViewExampleGroup#render given no input sends render(:template => (described file)) to the view
     Failure/Error: expect(view_spec.received.first).to eq([{:template => "widgets/new"},{}, nil])

       expected: [{:template=>"widgets/new"}, {}, nil]
            got: [{}, {}, nil]

       (compared using ==)

       Diff:
       @@ -1,2 +1,2 @@
       -[{:template=>"widgets/new"}, {}, nil]
       +[{}, {}, nil]
```

It does not make sense to me, I don't see any `prepend` coming into play.

Reply:
It must be something similar, as this exists as is I suggest leaving it.